### DISCUSSION TITLE
免费的连图都不能贴，不支持markdown？

### DISCUSSION THREAD

**Discussion by amiaoapp** [streetwriters/notesnook#919] [Q&A] [answered]
就因为开源？就成了香饽饽？1.9k的star?
不明白，joplin不好用吗？

连个基本功能都得收费，还那么贵？你们图的啥？

**Comment by thecodrr** [comment, marked as answer]
You don't get your food for free either. Sure, it'd be amazing if Notesnook offered everything for free but unfortunately, this is the way it works. To make Notesnook sustainable & progressive we have to charge a small fee (starting from $0.99/mo). If you cannot be bothered to pay then you can use the limited free version. There is literally no reason to complain.

Markdown is supported but not in it's raw form. You can write markdown and it converts to rich text on the fly. Raw markdown support is not advertised nor will be added. You can import your raw Markdown notes using the importer.
